exporters/prometheus-dcgm/dcgm-exporter/dcgm-exporter

```diff
@@ -72,16 +72,27 @@ function metric(name, type, help, value) {
         if (gpu == 0) {
             printf "# HELP dcgm_%s %s\n", name, help > out".swp"
             printf "# TYPE dcgm_%s %s\n", name, type > out".swp"
+
+            if (name == "device_count") {
+                printf "dcgm_%s %s\n", name, value > out".swp"
+            }
+        }
+
+        if (name != "device_count") {
+            printf "dcgm_%s{gpu=\"%s\",uuid=\"%s\",gpu_type=\"%s\"} %s\n", name, gpu, uuid, gpu_type, value > out".swp"
         }
-        printf "dcgm_%s{gpu=\"%s\",uuid=\"%s\"} %s\n", name, gpu, uuid, value > out".swp"
     }
 }
 (NF && NR > 2 && !($1 ~ "^#" || $1 ~ "^Id")) {
     # Labels
     i = 1
     gpu = $(i++)                                                                                                      # field 0 (implicit)
+    gpu_type = $(i++)"-"$(i++)                                                                                        # field 50
```

Comment:
```
dcgmi dmon -e 50
# GPU/Sw                 DVNAM
      Id
       0  Tesla V100-SXM2-16GB
       0  Tesla V100-SXM2-16GB
       0  Tesla V100-SXM2-16GB
       0  Tesla V100-SXM2-16GB
       0  Tesla V100-SXM2-16GB
       0  Tesla V100-SXM2-16GB
       0  Tesla V100-SXM2-16GB
```

Line of product and model are two separate fields. That's the reason we have two $(i++)